src/Sulu/Bundle/DocumentManagerBundle/Resources/config/behaviors.xml

```diff
@@ -62,6 +67,25 @@
             <tag name="sulu_document_manager.event_subscriber" />
         </service>
 
+        <service id="sulu_document_manager.subscriber.behavior.filing" class="%sulu_document_manager.subscriber.filing.class%">
+            <argument type="service" id="sulu_document_manager.node_manager" />
+            <argument>/%sulu.content.node_names.base%</argument>
+            <tag name="sulu_document_manager.event_subscriber" />
+        </service>
+
+        <service id="sulu_document_manager.subscriber.behavior.alias" class="%sulu_document_manager.subscriber.alias.class%">
+            <argument type="service" id="sulu_document_manager.node_manager" />
+            <argument type="service" id="sulu_document_manager.metadata_factory.base" />
+            <argument>/%sulu.content.node_names.base%</argument>
+            <tag name="sulu_document_manager.event_subscriber" />
+        </service>
+
+        <service id="sulu_document_manager.subscriber.behavior.reset.path" class="%sulu_document_manager.subscriber.behavior.reset.path.class%">
+            <argument type="service" id="sulu_document_manager.node_manager" />
+            <argument>/%sulu.content.node_names.base%</argument>
+            <tag name="sulu_document_manager.event_subscriber" />
+        </service>
```

Comment:
Shouldn't this `%sulu.content.node_names.base%` property only be necessary for the `ResetFilingBehavior`? Then I would also remove it from the `AbstractFilingBehavior`.